### ISSUE TITLE
8416 Staging - Remove default fieldset

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-team#122155]
## Description
I believe the forms library has a default set the wraps everything in a `<fieldset>` and I've been told there is a way to turn this off. There are pros and cons to this and the a11y folks talk about this a lot but more often than not, we have heard feedback that this markup makes things very noisy, especially when there are nested fieldsets (dates, radio groups, checkboxes). 

If we have the form set up properly (which hopefully we are) and the focus going to the H3s (which we will possibly post-MVP) there should be enough context to allow folks to know what info they need to add to the fields without the need for the overall fieldset on each form page.

### Acceptance criteria

- [ ] Turn off the default wrapping `<fieldset>` on the form pages

**Comment by meaghan-sullivan** [comment]
Needs triage to determine blocking status

**Comment by meaghan-sullivan** [comment]
Confirmed in staging! Closing under Iteration 7 since it is my fault that I didn't know how to test this

<img width="889" height="516" alt="Image" src="https://github.com/user-attachments/assets/825c1bbc-3e59-4009-ab82-73055f30c9a5" />

<img width="883" height="497" alt="Image" src="https://github.com/user-attachments/assets/73f3517b-6072-4ab0-ba8d-26b5b49baebc" />
